app/views/users/new.html.erb

```diff
@@ -0,0 +1,25 @@
+
+ <%= form_for(@user, as: :user, url: users_path, authenticity_token: form_authenticity_token)do |user|%>
+  <%= render 'shared/error_messages' %>
+  <%= user.label(:username, 'Username') %>
+  <%= user.text_field(:username)%>
```

Comment:
When I tried to render the new user page `/users/new` there was error `First argument in form cannot contain nil or be empty` meaning that the `@user` is nill.

HINT: you haven't created a new user in the `new` action to be available from the view file.